WORKSPACE

```diff
@@ -1,3 +1,15 @@
+http_archive(
+    name = "io_kubernetes_build",
+    sha256 = "8e49ac066fbaadd475bd63762caa90f81cd1880eba4cc25faa93355ef5fa2739",
+    strip_prefix = "repo-infra-e26fc85d14a1d3dc25569831acc06919673c545a",
+    urls = ["https://github.com/kubernetes/repo-infra/archive/e26fc85d14a1d3dc25569831acc06919673c545a.tar.gz"],
+)
+
+load("@io_kubernetes_build//defs:bazel_version.bzl", "check_version")
+# Ensure minimim supported bazel version
+# TODO(bentheelder): keep this up to date!
```

Comment:
this seems like an unsatisfiable TODO

Reply:
should I remove it then? someone should be updating these when we upgrade deps, we can pretty easily check which is minimum with planter?

Reply:
removed.